build.gradle

```diff
@@ -65,3 +66,7 @@ test {
         events "passed", "skipped", "failed"
     }
 }
+
+jar {
```

Comment:
Low: Why removing the jar creation?

Reply:
This just prevents the plain/slim jar being created which confuses drone otherwise
https://docs.spring.io/spring-boot/docs/2.5.1/gradle-plugin/reference/htmlsingle/#packaging-executable.and-plain-archives